tensorflow/core/kernels/fake_quant_ops_functor.h

```diff
@@ -88,7 +88,7 @@ struct FakeQuantWithMinMaxArgsFunctor {
     Nudge(min, max, quant_min, quant_max, &nudged_min, &nudged_max,
           &nudged_scale);
 
-    const float inv_nudged_scale = 1.0f / nudged_scale;
+    const float inv_nudged_scale = (quant_max - quant_min) / (max - min);
```

Comment:
Computing this here risks being out of sync with `nudge_scale` if its computation is updated in the future. Please modify the `Nudge` function to also return `inv_nudge_scale` instead. Please also add a comment there that `inv_nudge_scale` is computed separately to preserve precision.